core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -95,28 +98,31 @@ public void checkEnabled() {
     }
 
     private void updateCache() {
-        activeTriggers.clear();
```

Comment:
This was the problem, triggers were not run between this call and adding them all back later on.